### ISSUE TITLE
Guide refresh

### ISSUE THREAD

**Issue by andrewda** [andrewda/node-steam-guide#100]
Hello everyone!

I started writing this guide a few years ago because I noticed there was a lack of quality information out there about how to create Steam bots. Sure, everyone and their dog can find guides online about creating Slack bots and Discord bots in Node.js, but there wasn't any in-depth guide on how to navigate the oddities of Steam bot programming.

Over the past few years, interest and traffic to this guide has varied quite drastically, but over the past few months there seems to be a spike in demand. I've noticed this not only in this repository's insights, but also in the number of people who have contacted me about questions or job offers.

Given that Steam bots seem to once again be on the rise coming into this new decade, I've decided that it would be a good idea to do a complete refresh of this guide, from the ground up.

This guide is meant to help _you_, so I'd really appreciate some suggestions into what kind of things you'd like to see in this new guide. I feel like the best way of learning is by getting your hands dirty and building something, so suggestions about what to build are welcome (nothing against Steam's TOS, like gambling websites, please!)

Finally, I am incredibly grateful for [everyone who has contributed](https://github.com/andrewda/node-steam-guide/graphs/contributors) to this guide over the past few years -- it would not be the same without you!

_If this guide has helped you at all, please consider [sponsoring me on GitHub](https://github.com/sponsors/andrewda)! Writing this guide takes a lot of time and effort, so sponsoring me will help ensure that I have as much time as possible to get this new guide written. I totally understand if you don't have the funds to donate, so luckily there are other ways to help the project, like creating an issue or submitting a PR!_

Best,
Andrew

------

## Things I'd like to hear from you

- Database to use? MongoDB? MySQL?
- Project to do? Trading website (like BitSkins)? Selling website (like csgo.cash)?

**Comment by jeanmichelmorin** [comment]
Yes this is a great idea. I'd really like to see MySQL instead of MongoDB.

**Comment by andrewda** [comment]
@jeanmichelmorin Thanks for the suggestion!

**Comment by jeanmichelmorin** [comment]
Also, do you think you could do videos? I think videos are way easier to follow because you can explain everything step by step as you do the code.

**Comment by andrewda** [comment]
Interesting idea, I hadn't considered that. I might write the written guide first, and look into publishing videos after since I expect they would take a significant amount of time.

**Comment by Xdeposh** [comment]
Would it be possible to go through the creation of a steam leveling website?
Also as mentioned previously, videos would help immensely!
Cheers♥

**Comment by dezepmup** [comment]
Hello, I'm really looking forward to the update, I would like to see MySQL and Selling site (like csgo.cash), can i help with it?

**Comment by dezepmup** [comment]
can i help with it*
this
